docs/conversion_examples_gallery/behavior/video.rst

```diff
@@ -86,3 +86,79 @@ If using an older version of neuroconv (<0.8), you can use the :py:class:`~neuro
     >>> # Choose a path for saving the nwb file and run the conversion
     >>> nwbfile_path = f"{path_to_save_nwbfile}"  # This should be something like: "saved_file.nwb"
     >>> interface.run_conversion(nwbfile_path=nwbfile_path, metadata=metadata, overwrite=True)
+
+
+Specifying Metadata
+~~~~~~~~~~~~~~~~~~~
+
+The examples above show how to convert video data without specifying any metadata, in which case the metadata will be
+automatically generated with default values. To ensure that the NWB file is properly annotated, specify the metadata
+using the formats described below.
+
+For :py:class:`~neuroconv.datainterfaces.behavior.video.externalvideodatainterface.ExternalVideoInterface`,
+use the following structure:
+
+.. code-block:: yaml
+
+    # external_video_metadata.yaml
+    Behavior:
+      ExternalVideo:
+        MyExternalVideo:
```

Comment:
We have to communicate somehow that ExternalVideo is also the `video_name` at the `__init__` of the interface.

Reply:
I think it's communicated adequately. In the code you can see that `video_name="MyExternalVideo"`, and the default (`video_name="ExternalVideo"`) is communicated in the docstring/API docs.

Reply:
I think that a comment in the yaml saying that this is the same as video_name on the init will be an improvement. 

The vide_name is indeed in the code but I know is there because I already know. The docstring is not available in the code either.